encoding.go

```diff
@@ -70,6 +83,9 @@ var encodingMap map[string]*Encoding
 var l *sync.Mutex
 
 func init() {
+	gob.NewDecoder(bytes.NewReader(cl100k)).Decode(&vocabCL100K)
```

Comment:
Why not just put this into each function? ex: cl100k_base() ...

Reply:
because it would be inefficient to have to re-decode the gob every time